index.html

```diff
@@ -81,235 +81,215 @@
-      removed from the document.</p>
-      <pre class="example">
-  Link: &lt;https://example.com/font.woff&gt;; rel=preload; as=font
-  Link: &lt;https://example.com/app/script.js&gt;; rel=preload; as=script
-  Link: &lt;https://example.com/logo-hires.jpg&gt;; rel=preload; as=image
-</pre>
-      <pre class="example highlight js">
-  &lt;link rel="preload" href="//example.com/widget.html" as="iframe"&gt;
-</pre>
-      <pre class="example highlight js">
-  &lt;script&gt;
-  var res = document.createElement("link");
-  res.rel = "preload";
-  res.as = "iframe";
-  res.href = "/other/widget.html";
-  document.head.appendChild(res);
-  &lt;/script&gt;
-</pre>
-      <p>The appropriate times to <a href=
-      "http://www.w3.org/TR/html5/document-metadata.html#concept-link-obtain">obtain
-      the resource</a> are:</p>
-      <ul>
-        <li>When the user agent that supports [[!RFC5988]] processes <a>preload
-        link</a>'s specified via the <a href=
-        "http://tools.ietf.org/html/rfc5988">Link HTTP header</a>.
+        <li>When the <a>preload link</a> is created on a [link] element that is
+        already [in a Document].
         </li>
-        <li>When the <a>preload link</a>'s link element is <a href=
-        "http://www.w3.org/TR/html5/infrastructure.html#in-a-document">inserted
-        into a Document</a>.
+        <li>When the `href` attribute of the [link] element of a <a>preload
+        link</a> that is already [in a Document] is changed.
         </li>
-        <li>When the <a>preload link</a> is created on a `link` element that is
-        already <a href=
-        "http://www.w3.org/TR/html5/infrastructure.html#in-a-document">in a
-        Document</a>.
+        <li>When the `crossorigin` attribute of the [link] element of a
+        <a>preload link</a> that is already [in a Document] is set, changed, or
+        removed.
         </li>
-        <li>When the `href` attribute of the `link` element of an <a>preload
-        link</a> that is already <a href=
-        "http://www.w3.org/TR/html5/infrastructure.html#in-a-document">in a
-        Document</a> is changed.
+        <li>When the `type` attribute of the [link] element of a <a>preload
+        link</a> that is already [in a Document] is set or changed to a value
+        that does not or no longer matches the [Content-Type metadata] of the
+        previous obtained external resource, if any.
         </li>
-        <li>When the `crossorigin` attribute of the `link` element of a
-        <a>preload link</a> that is already <a href=
-        "http://www.w3.org/TR/html5/infrastructure.html#in-a-document">in a
-        Document</a> is set, changed, or removed.
+        <li>When the `type` attribute of the [link] element of a <a>preload
+        link</a> that is already [in a Document] but was previously not
+        obtained due to the `type` attribute specifying an unsupported type is
+        set, removed, or changed.
```

Comment:
s/set, removed or changed/removed or changed to an attribute specifying a supported type/ ?
If the element was in the document but not obtained, he could have had a previous `type` attribute, right?